_extras/motivation.md

```diff
@@ -2,5 +2,46 @@
 title: Motivation
 ---
 
+# Cross Stitching is like computer programming
+
+Computer programming or coding requires analytical skills, which allows us to think of a solution to a problem, design a plan comprising of multiple steps and accomplish them by writing multiple lines of code. 
+Each of these lines have been intentionally written to play a vital role in a working software.
+
+Cross stitching is quite like programming where you create a beautiful pattern by "writing" in rows. 
+If there’s a bug or mis-stitching in a row, you have to fix that to create your pattern as you intented to look like.
+
+Keeping that in mind, we invite you to explore cross-stitching as a hobby for programmers. 
+This lesson is designed to train a novice learner to learn how to cross stitch and use it as a hobby to mindfully enjoy their breaks from work.
+
+# Why cross stitch as a programmer?
+
+On a daily basis, we use smartphone and computers for everything we do. 
+Starting from getting a fast access to news and latest information in the morning, to learning a new programming language or other skills to getting our job done at work, to ordering things that makes our day-to-day life possible.
+Needless to say, such excessive use of technology leads to information overload, and often results into burnout and other mental health issues including anxiety, and depression [1].
+
+<SARAH HELP ME>
+
+# Mindful Hobbies in Digital Age
+
+> *Mindfulness and technology is a movement in research and design, that encourages the user to become aware of the present moment, rather than losing oneself in a technological device. This field encompasses multidisciplinary participation between design, psychology, computer science, and religion.* Source: [Wikipedia](https://en.wikipedia.org/wiki/Mindfulness_and_technology)
+
+Mindfulness has been theoretically and empirically associated with psychological well-being [2]. 
+Practicing mindfulness through a habby is an easy way to intentionally separate our 'down-time' from 'work-time' and boost our wellbeing. 
+It is also recommended by National Institute for Health and Care Excellence to boost attention and concentration, and to prevent recurrent depression [3].
```

Comment:
```suggestion
The [National Institute for Health and Care Excellence](INSERT LINK) recommends mindfulness as a way to boost attention and concentration and to prevent recurrent depression [3].
```